docs_dev/tf_serving.md

```diff
@@ -0,0 +1,72 @@
+# TF Serving in Kubeflow
+
+Authors:
+
+- lunkai@google.com
+
+Status:
+
+- 2018-07-10 Created
+
+## TF Serving
+[Tensorflow Serving](https://github.com/tensorflow/serving)
+is a high-performance serving system for ML models.
+
+It provides gRPC [apis](https://github.com/tensorflow/serving/tree/master/tensorflow_serving/apis)
+for prediction and model service.
+
+Starting in TF 1.8, TF Serving also provides a REST [api](https://github.com/tensorflow/serving/blob/master/tensorflow_serving/g3doc/api_rest.md).
+
+## Kubeflow TF Serving component
+
+In Kubeflow, we have a TF Serving [component](https://github.com/kubeflow/kubeflow/tree/master/kubeflow/tf-serving).
+
+### HTTP proxy
+The [http proxy](https://github.com/kubeflow/kubeflow/tree/master/components/k8s-model-server/http-proxy)
+provides an REST api, which is the same as TF Serving's built-in REST api.
+
+We initially implemented this because TF Serving only supported gRPC api (until
+TF 1.8). Going forward, we will still maintain and support this http proxy as
+it has the ability to provide additional features such as:
+- exporting metrics, [kubeflow/kubeflow#1036](https://github.com/kubeflow/kubeflow/issues/1036).
+- request logging. See
```

Comment:
Are we planning on doing request logging in the http proxy? Why can't we do that in TF Serving?
Would we have to also proxy gRPC requests in order to get that functionality when gRPC is used?

Reply:
I think it's easier to make progress if we do it in our http proxy comparing to TF serving.

We can discuss on this thread https://github.com/kubeflow/kubeflow/issues/1000

Reply:
Can you add a link to the issue in the doc?

Reply:
It's there